support-frontend/test/controllers/ApplicationTest.scala

```diff
@@ -38,7 +44,19 @@ class ApplicationTest extends AnyWordSpec with Matchers with TestCSRFComponents
   )
 
   "/healthcheck" should {
+
     "return healthy" in {
+      val priceSummaryServiceProvider = {
+        val priceSummaryService = mock[PriceSummaryService]
+        when(
+          priceSummaryService
+            .getPrices[SupporterPlus.type](any[SupporterPlus.type], any[List[PromoCode]], any[ReaderType]),
+        )
+          .thenReturn(Map[CountryGroup, CountryGroupPrices](CountryGroup.UK -> Map.empty))
+        val priceSummaryServiceProvider = mock[PriceSummaryServiceProvider]
+        when(priceSummaryServiceProvider.forUser(anyBoolean())).thenReturn(priceSummaryService)
+        priceSummaryServiceProvider
+      }
```

Comment:
I updated the test so it's still testing the happy case, but I didn't actually add a test for heathcheck failing, not sure if it's necessary